bin/truffleruby

```diff
@@ -99,6 +122,9 @@ if [ $on_graalvm = false ]; then
 
 fi
 
+# no " to split $JAVA_OPTS into array elements
+java_args+=($JAVA_OPTS)
```

Comment:
having two variables with java options makes it more complicated and prone to errors, e.g.  -cmd was previously ignored in JAVA_OPTS.